### DISCUSSION TITLE
LosslessCut runs dreadfully slow for me

### DISCUSSION THREAD

**Discussion by CaseyStelken** [mifi/lossless-cut#2024] [Support]
I'm not sure what the problem is here.  I'm really excited to use LosslessCut after everything I've read about it, but it is just not liking my system for some reason.  I'm running Windows 10 home 64-bit with an AMD Ryzen 7 eight-core processor and a NVIDIA GeForce GTX 1060 6GB graphics card.

I don't normally have trouble running simple software like this.  But the program takes quite awhile to open, or to load a video file of any size, or to play back the file... basically every operation takes ages to respond, and I constantly get the old "(Not Responding)" line added to the title bar until it can figure out what it's doing.

I tried installing an older version, just to see if it would make a difference... no difference.

Any suggestions would be MUCH appreciated!

**Comment by mifi** [comment]
Hi. I'm not sure what's causing this. is Chrome also slow for you by any chance (if you open the same video in chrome, does it also play slowly?) Did you have a chance to test it on a different computer? for me on m2 macbook air it's lightning fast

could also be caused by some antivirus software, maybe you could try to temporarily disable and see if it helps
